docs/src/guide/interceptors.md

```diff
@@ -36,3 +36,20 @@ axios.interceptors.request.use((req) => req);
 // This will be ran BEFORE the cache interceptor
 axios.interceptors.response.use((res) => res);
 ```
+
+---
+
+## Extending types
+
```

Comment:
What about: 

> When using axios-cache-interceptor, you will note that it have a different type than the defaults `AxiosInstance`, `AxiosRequestConfig` and `AxiosResponse`. That's because was chosen to override axios's interfaces instead of extending, to avoid breaking changes with other libraries.
>
> However, this also means that, when integrating with other packages or creating your own custom interceptor, you need to override/extend our own types, `CacheInstance`, `CacheRequestConfig` and `CacheAxiosResponse` to match your needs.
>
> This can be done as shown below:
>
> ```ts
> declare module 'axios-cache-interceptor' {
>    interface CacheRequestConfig<R = unknown, D = unknown> {
>       property?: number
>    }
> }
>
> // Now it has type checking and autocompletion!
> axios.get({ property: 123 })
> ```

Reply:
Looks great to me! Thanks